### PR TITLE
broken mailto link - Looking for help! <ARRAY(0xcde66a0)>

### LINKED ISSUE #2232: broken mailto link - Looking for help! <ARRAY(0xcde66a0)>
In some /pod endpoints the following -  pod_file.module[0].name - looks to be returning as an array.

Example - https://metacpan.org/pod/distribution/Memcached-libmemcached/lib/Memcached/libmemcached/constants.pm

### PR DESCRIPTION
**Pull request by ThisUsedToBeAnEmail**
In template toolkit it is array.0.key and not array[0].key

Closes #2232

### CODE REVIEW

File: root/inc/notification.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
             has_email  = 1,
             title  = "Why not adopt me?",
             text   = "This distribution is up for adoption! If you're interested then please contact pause module admins via ",
-            body   = format("Hey %s,\r\n\r\nI'm interested in adopting the %s module. If you can grant me the privilidge I will release the next version. Kind Regards,"),
-            email  = author.email,
-            name   = author.name
+            body   = format("Hey %s,\r\n\r\nI'm interested in adopting the %s module. If you can grant me the privilidge I will release the next version.\r\n\r\n Kind Regards,"),
```
Comment (oalders) [inline root/inc/notification.html]: s/privilidge/privilege/

**Comment by oalders** [comment]
My one concern is that we could have caught this with a unit test.  If we really wanted to, we could upload an `ACME` dist with the various permissions that we're trying to test.  Not saying that needs to be done here, but it would be nice to be able to work on this with confidence in the future, knowing that syntax errors in the templates will get caught.

**Comment by ThisUsedToBeAnEmail** [comment]
It's lucky I'm not looking for a Job.

**Comment by oalders** [comment]
There are a couple of other typos in the file, but not in the diff that could be fixed:

`If you're intereset`=> `If you're interested`
`privilidge` comes up twice
`contact pause module admins` => `contact the PAUSE module admins`

Thanks for all of your work on this.  It's greatly appreciated.  I apologize for not catching these on the first pass.

**Comment by oalders** [comment]
Thanks.  Are you able to fix those other typos?

**Comment by ThisUsedToBeAnEmail** [comment]
I asked for the text to be provided to me for a reason.

**Pull request merged**